recipe/meta.yaml

```
@@ -56,38 +35,22 @@ requirements:
   build:
     - {{ compiler('c') }}
     - {{ compiler('cxx') }}
+    - python                              # [build_platform != target_platform]
+    - cross-python_{{ target_platform }}  # [build_platform != target_platform]
     - git
     - ninja
+    - ccache
     - pkg-config # to generate openvino.pc on macOS
-    - scons  # [aarch64 or arm64]
     - cmake >=3.15 # to have new cmake cmd interface (-S, --install and other options)
-    - python >=3.7
-    - libprotobuf =3.20.3 # for protoc compiler, libraries are used from libprotobuf-static
-    - flatbuffers =23.3.3
     - sysroot_linux-64 =2.17  # [linux64]
   host:
```

You need to list Python in host.
```suggestion
  host:
    - python
```

---

Thanks for your suggestion! I have these requirements, but for specific output (libopenvino-python)

We have multiple outputs in this recipe, while only one of them depend on different python versions.
What is the proper way? 
- (a) Rebuild all outputs for each python versions
- (b) Rebuild only specific output which depend on the python.
I suppose if I specify python as a `host` requirement globally, CI will rebuild everything from scratch (variant (a) ). But is possible to rebuild only specific output? (variant (b) ). Is it any approach how to control which output to rebuild?

---

You should only need it on the Python specific outputs. xgboost should be a good example on how it is handled.